addons/website/static/tests/interactions/snippets/countdown.test.js

```diff
@@ -91,31 +90,63 @@ test("countdown interaction update the canvas for seconds correctly", async () =
     const canvasSeconds2 = canvass2[3];
     const dataSeconds2 = canvasSeconds2.getContext('2d').getImageData(0, 0, canvasSeconds2.width, canvasSeconds2.height).data;
 
+    await delay(1000);
+
+    // time T + 2s
+
+    const canvass3 = el.querySelectorAll('canvas');
```

Comment:
`canvass1/2/3` ?? Why not `canvas` ? (just wondering)

Oh ! It's the plural ! I guess that should be `canvases`, but I'd rather go for `canvas3Els`. As you wish.